tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -28,16 +28,12 @@ public function testDiffReturnsDiff()
         $diff = <<<'TXT'
 --- Original
 +++ New
-@@ @@
+@@ -1,3 +1,4 @@
  <?php
  class Foo extends Bar {
 -    function __construct($foo, $bar) {
 +    public function __construct($foo, $bar)
 +    {
-         $this->foo = $foo;
```

Comment:
I didn't mean to show all the lines after the changes (which was indeed the reason of my commit you mentioned and should not happen), but only a few, like e.g.:

File A.txt:
```
A
B
C
D
E
F
G
H
I
```

File B.txt:
```
A
B
C
D

F
G
H
I
```

Diff:
```
 $ diff -u A.txt B.txt
--- A.txt	2017-08-23 19:34:19.194610864 +0200
+++ B.txt	2017-08-23 19:34:11.984558683 +0200
@@ -2,7 +2,7 @@
 B
 C
 D
-E
+
 F
 G
 H
```

This diff shows 3 unchanged lines before the changed one (skipping the first one) and 3 after (skipping the last one). Isn't that possible with the new differ?